irods/test/helpers.py

```diff
@@ -151,6 +151,18 @@ def recast(k):
 # Create a connection for test, based on ~/.irods environment by default.
 
 def make_session(test_server_version = True, **kwargs):
+    '''
+    Connect to an active iRODS server as determined by any client environment
```

Comment:
The word, active, seems unnecessary. Is something special being conveyed by its use?

Reply:
RIght , it isn't necessary

Reply:
Removed it.